opal/mca/common/ucx/common_ucx_wpool.c

```diff
@@ -215,11 +215,9 @@ opal_common_ucx_wpool_init(opal_common_ucx_wpool_t *wpool,
 err_wpool_add:
     free(wpool->recv_waddr);
 err_get_addr:
-    if (NULL != wpool) {
-        OBJ_RELEASE(winfo);
-        OBJ_RELEASE(wpool->dflt_winfo);
-        wpool->dflt_winfo = NULL;
-    }
+    OBJ_RELEASE(winfo);
+    OBJ_RELEASE(wpool->dflt_winfo);
+    wpool->dflt_winfo = NULL;
  err_worker_create:
     ucp_cleanup(wpool->ucp_ctx);
```

Comment:
```suggestion
    OBJ_DESTRUCT(&wpool->active_workers);
    OBJ_DESTRUCT(&wpool->active_workers);
    ucp_cleanup(wpool->ucp_ctx);
```

Reply:
Typo? (one should be `idle_workers`)

```suggestion
    OBJ_DESTRUCT(&wpool->idle_workers);
    OBJ_DESTRUCT(&wpool->active_workers);
    ucp_cleanup(wpool->ucp_ctx);
```

Reply:
updated PR